OpenSim/Simulation/Control/PrescribedController.h

```diff
@@ -58,7 +56,8 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PrescribedController, Controller);
     /** (Optional) prescribed controls from a storage file  */
     OpenSim_DECLARE_OPTIONAL_PROPERTY(controls_file, std::string,
         "Controls storage (.sto) file containing controls for individual "
-        "actuators in the model. Column labels must match actuator names.");
+        "actuators in the model. Column labels must match actuator names or "
+        "absolute paths.");
```

Comment:
I'd clarify "absolute paths" as "full path names of actuators in the model"? From the code it appears that mix and match is allowed but may not be necessary to document (match either actuator names or  full path names of actuators in the model)

Reply:
Updated. I chose `absolute path` for consistency with `Component::getAbsolutePath()`.